src/com/akshay/captain/jira.main.kts

```diff
@@ -0,0 +1,240 @@
+@file:DependsOn("org.slf4j:slf4j-simple:1.7.24")
+@file:Repository("https://repo1.maven.org/maven2/")
+@file:Repository("https://repo.maven.apache.org/maven2")
+@file:Repository("https://packages.atlassian.com/maven/repository/public")
+@file:DependsOn("com.slack.api:slack-api-client:1.11.0")
+@file:DependsOn("net.rcarz:jira-client:0.5")
+@file:DependsOn("com.squareup.okhttp3:okhttp:4.9.3")
+
+import Jira_main.Platform
+import Jira_main.TicketType
+import net.rcarz.jiraclient.BasicCredentials
+import net.rcarz.jiraclient.Issue
+import net.rcarz.jiraclient.IssueType
+import net.rcarz.jiraclient.JiraClient
+import net.sf.json.JSONObject
+import okhttp3.*
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.RequestBody.Companion.toRequestBody
+import java.io.IOException
+
+
+//region Configuration
+private val changeEmoji = "✨"
+private val techChangesEmoji = "🤖"
+private val bugEmoji = "🐛"
+private val instrumentationEmoji = "📈"
+//endregion
+
+//region constant
+private val changes = "Changes"
+private val techChanges = "Tech changes"
+private val bugFixes = "Bug fixes"
+private val instrumentation = "Instrumentation"
+
+//Jira Variables
+private val clueJiraUrl = "https://biowink.atlassian.net"
+private val readyForReleaseFilter = "filter=15768"
+private val firstIndex = 0
+private val authorization = "Authorization"
+private val jsonType = "application/json; charset=utf-8"
+private val versionEndPoint = "/rest/api/2/version"
+
+//Platform
+private val android = "android"
+private val ios = "ios"
```

Comment:
Could we use the sealed class Platform to get this values?

Reply:
have one sealed class for Platform. Having for string made no sense